processing/data_collection/gazette/pipelines.py

```diff
@@ -19,7 +19,7 @@ def process_item(self, item, spider):
 
     def pdf_source_text(self, item):
         pdf_path = os.path.join(FILES_STORE, item['files'][0]['path'])
-        command = f'pdftotext -layout "{pdf_path}"'
+        command = f'pdftotext -layout {pdf_path}'
```

Comment:
Just out of curiosity: why removing these quotes? Does `subprocess.run` auto-escape characters such as spaces?

Reply:
I had added this quotes to "solve the problem" mentioned in the issue #15. But I had not seen that there was already a discussion and a PR(#28) about the problem. I believe that _subprocess.run_ does not perform autoescape, but the PR guarantees a valid pathname.

Reply:
My point is that in Linux, for example, `/tmp/My file.pdf` is a _valid_ path name. However in the command line either you'll have to escape it (`pdftotext -layout /tmp/My\ file.pdf`) or to use quotes (`pdftotext -layout "/tmp/My file.pdf"`).

What I'm trying to say is that quotes might be good where they are ; )